src/Helper/Quote/Customer.php

```diff
@@ -123,16 +123,22 @@ public function setAddress(MageQuote $quote)
     {
         $billing = $this->sgBase->getInvoiceAddress();
         if (!empty($billing)) {
-            //$quote->getBillingAddress()->setData('should_ignore_validation', true); //todo-sg: cannot find check
-            $data = $this->sgCustomer->createAddressData($this->sgBase, $quote, $billing);
-            $quote->getBillingAddress()->addData($data)->setCustomerAddressId($billing->getId());
+            $data = $this->sgCustomer->createAddressData($this->sgBase, $billing, true);
```

Comment:
@aaron-martin & @Skistims please double check this signature. This code cannot work as it throws an exception when passed to `createAddressData` function. E.g. `true !== \ShopgateAddress`

Reply:
I already reaized that and commented it in the JIRA ticket.
there's also a not working call in the import plugin... not sure how this inconsistency came up, but I will look into it before releasing.
Thanks for letting me know anyway